lib/node/remix.ts

```diff
@@ -26,6 +43,17 @@ export function getRoutes(options: Options) {
     };
   }
 
+  if (options.routes) {
```

Comment:
Options are destructured at the beginning of this function. I would do the same for `option.routes`.

Reply:
I'd initially did that because there was already a `routes` variable defined down below. I swapped out the variable.